akka-http/src/main/scala/akka/http/scaladsl/server/directives/SecurityDirectives.scala

```diff
@@ -302,17 +302,31 @@ object Credentials {
      * See also [[EnhancedString#secure_==]], for more information.
      */
     def verify(secret: String): Boolean = verify(secret, x => x)
+
+    /**
+     * Compares with custom 'verifier' the received secret part of the Credentials.
+     * Use of this method only if custom String equality testing is required not recommended.
```

Comment:
Not sure if I parse this sentence correctly, is it:
```suggestion
     * Use of this method only if custom String equality testing is required, not recommended.
```
?